argparse_test.go

```diff
@@ -1362,7 +1684,7 @@ func TestUsageHidden1(t *testing.T) {
 	_ = cmd1.Flag("a", "verylongflagA", &Options{Required: true, Help: "flag1 description"})
 	_ = p.String("s", "verylongstring-flag1", &Options{Help: "string1 description"})
 	_ = p.Int("i", "integer-flag1", &Options{Help: "integer1 description"})
-	_ = p.Int("i2", "integer-flag2", &Options{Help: DisableDescription})
```

Comment:
Description disabled because of "i2"  - long shorthand argument and silent drop of argument, not because of DisableDescription.